gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EcsCloudMetricController.groovy

```diff
@@ -0,0 +1,21 @@
+package com.netflix.spinnaker.gate.controllers
```

Comment:
Please put that in the ecs package, just like AmazonInfrastructureController is in its own package

Reply:
Done.

Also carried `EcsClusterController` over into the new package.